### PR TITLE
Date time form fixes

### PR DESCRIPTION
**Pull request by FriedSock**

### CODE REVIEW

File: app/views/defence_requests/form_partials/_detainee_date_of_birth.html.erb [hunk 1]
```diff
@@ -7,17 +7,17 @@
     <%= f.fields_for :date_of_birth, @defence_request_form.fields[:date_of_birth] do |t| %>
       <div class="date-field-wrapper">
         <%= t.label t('day'), class: "form-label" %>
-        <%= t.text_field :day, size: 2, placeholder: 'DD', class: 'day text-field not-given-check',
```
Comment (robmckinnon) [inline app/views/defence_requests/form_partials/_detainee_date_of_birth.html.erb]: Has placeholder text been removed as part of the design?

Comment (FriedSock) [inline app/views/defence_requests/form_partials/_detainee_date_of_birth.html.erb]: yes

Comment (ansonK) [inline app/views/defence_requests/form_partials/_detainee_date_of_birth.html.erb]: Yes. No placeholders in date / date-time fields

**Comment by robmckinnon** [comment]
Note: as discussed verbally, this PR breaks the "Today/Tomorrow" design for Solicitor entering interview time. One approach to fix that would be to make the toggles shown by the DateChooser js configurable.

**Comment by FriedSock** [comment]
It was already broken, the design is "Today/Tomorrow/Day after Tomorrow"

**Pull request merged**